kafka/web/README.md

```diff
@@ -0,0 +1,22 @@
+# Kafka Web App for Confidential Containers on Azure
+
+## About Web App
+
+This is based on the Next.js app starter that contains a frontend in React and backend.
+Every five seconds the frontend calls the backend endpoint /api/data and is returned:
+
+```json
+{
+  "message":"<your-kafka-message-here>"
+}
+```
+
+Where the message is either encrypted (when running without a security policy in a Confidential Container environment or on a non-confidential environment)
+
+## Instructions
+
+Build OCI image of the web app by running: `docker build -t <image-name>:<tag> .`
+Push image to Azure Container Registry:
+`az acr login -n <registry-name>`
+`docker push <image-name>:<tag>`
+Deploy application with `kubectl`
```

Comment:
Where's the yaml?

Reply:
I was thinking Steven would put it along with his consumer/producer

Reply:
I would state something to that effect in the README just so that people know where to find it.

Reply:
I'll make an update to the docs after his parts of the example get merged. That way I can put an exact path instead of "it's somewhere else in this repo"